packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js

```diff
@@ -38,12 +38,49 @@ export class CheckoutDeliveryOption extends PureComponent {
         optionSubPrice: 0
     };
 
+    state = { isOverflowed: false };
+
+    rowRef = createRef();
+
+    subPriceRef = createRef();
+
+    componentDidMount() {
+        if (this.rowRef.current && this.subPriceRef.current) {
```

Comment:
This if block can be replaced with `updateState()` function call, as they do the same thing.